### DISCUSSION TITLE
Authenik and PartDB

### DISCUSSION THREAD

**Discussion by pixbalance** [Part-DB/Part-DB-server#595] [General]
Is anybody using PartDB with Authenik for SSO and SAML? I tried to transfer the information from Keycloak to Authentik (https://docs.part-db.de/installation/saml_sso.html)

I have partdb, authentik and NGINX installed on unraid as docker.
My domain is https://partdb.mydomain.de, Login with local credentials in PartDB is working.

**Current status:**
When I click on the SSO login button on https://partdb.mydomain.de I end up in a error 500 page.
When I open Authentik https://auth.mydomain.de, log in and click on the App Icon for PartDB, login is working and a user is created.
Although I have added the user attributes email, lastName and firstName the user is created with a random number and no information used.
When I am logged in into PartDB and try to logout, I end up at error 500 page.

I have made the following settings:

**SAML_ENABLED:**
1

**SAML_SP_ENTITY_ID:**
https://partdb.mydomain.de
this value is from the provider metadata attribute "entityID"

**SAML_IDP_ENTITY_ID:**
https://partdb.mydomain.de
same as SAML_SP_ENTITY_ID

**SAML_IDP_SINGLE_SIGN_ON_SERVICE:**
https://partdb.mydomain.de/application/saml/partdb/sso/binding/post/
this value is from the provider metadata attribute SingleLogoutService -> Location with Binding Http-Post

**SAML_IDP_SINGLE_LOGOUT_SERVICE:**
https://partdb.mydomain.de/application/saml/partdb/slo/binding/post/
this value is from the provider metadata attribute SingleSignOnService -> Location with Binding Http-Post

**SAML_IDP_X509_CERT:**
....
this value is from the provider metadata attribute ds:X509Certificate

**DEFAULT_URI:**
https://partdb.mydomain.de/

**SAML_ROLE_MAPPING:**
{"admin": 1,  "family": 5, "guest": 4, "*": 2}

**Comment by jbtronics** [comment]
What does Part-DB log says? There should be some error message saying what caused the 500 error.
